### PR TITLE
1.17.x displaytest backport

### PR DESCRIPTION
**Pull request by SrRapero720**
depends on #9841

### CODE REVIEW

File: mdk/src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -59,4 +67,4 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed mollis lacinia magn
 # This version range declares a minimum of the current minecraft version up to but not including the next major version
     versionRange="[@MC_VERSION@,@MC_NEXT_VERSION@)"
     ordering="NONE"
-    side="BOTH"
+    side="BOTH"
```
Comment (PaintNinja) [inline mdk/src/main/resources/META-INF/mods.toml]: Your IDE is removing line endings again :/

**Review by PaintNinja — CHANGES_REQUESTED** [review]

File: mdk/src/main/resources/META-INF/mods.toml (hunk 1 above)
Comment (SrRapero720) [reply mdk/src/main/resources/META-INF/mods.toml]: is a cherry pick from 1.18.2

Comment (SrRapero720) [reply mdk/src/main/resources/META-INF/mods.toml]: which is a cherry pick from 1.19.2

**Comment by Jonathing** [comment]
It looks like this PR has gone stale. Are there plans to resume working on this?

**Comment by LexManos** [comment]
Backporting the toolchain changes will eventually happen. But honestly its going to wait until my new toolchain system is done. So ya closing this for the time being.

**Pull request closed without merging**